UnitystationLauncher/UnitystationLauncher.csproj

```diff
@@ -109,6 +111,7 @@
     <PackageReference Include="runtime.any.System.Threading.Timer" Version="4.3.0" />
     <PackageReference Include="runtime.unix.System.Private.Uri" Version="4.3.2" />
     <PackageReference Include="Dotnet.Bundle" Version="0.9.13" />
+    <PackageReference Include="Microsoft.ILVerification" Version="8.0.0" />
```

Comment:
Looks like this line got duplicated when resolving the conflicts, this package is already up on line 74 / 76 in this file.

Reply:
strange, must be a part of rebase. I removed the redundant line. :)